packages/react-native/sdks/hermes-engine/hermes-utils.rb

```diff
@@ -298,14 +298,28 @@ def resolve_url_redirects(url)
 
 # [macOS react-native-macos does not publish macos specific hermes artifacts
 # so we attempt to find the latest patch version of the iOS artifacts and use that
```

Comment:
Update this comment, along with maybe the block below "versionWithoutPatch"?

Reply:
Done.